backend/models/Signer.js

```diff
@@ -11,6 +11,9 @@ const signerSchema = new Schema({
     type: Number,
     default: 2, // 5 = admin, 4 = moderator, 3 = contributor, 2 = member, 1 = guest
   },
+  email: String, //TODO MAYBE THIS SHOULD BE UNIQUE
```

Comment:
Could be. Arguably in `models/Project.js` as well.

Reply:
That would block people from creating multiple projects with the same email :(